atomicapp/cli/main.py

```diff
@@ -165,7 +165,7 @@ def set_arguments(self):
         parser_run.add_argument(
             "--provider",
             dest="cli_provider",
-            choices=['docker', 'kubernetes', 'openshift'],
+            choices=['docker', 'kubernetes', 'openshift', 'marathon'],
```

Comment:
Can we replace these list with a constant rather than having to update them on 2 places (run and stop)